pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -1339,6 +1375,40 @@ PicklistOption:
       type: string
     exclusive:
       type: boolean
+MatrixOption:
+  type: object
+  required:
+    - stableId
+    - optionLabel
+    - exclusive
+    - groupId
+  properties:
+    stableId:
+      type: string
+    groupId:
+      type: string
+      nullable: true
+    optionLabel:
+      type: string
+    tooltip:
+      type: string
+      description: tooltip for the option, in plain text
+    exclusive:
+      type: boolean
+      description: if `exclusive` option selected, other options will be cleared (applicable for MULTIPLE select mode)
```

Comment:
Does this de-selection cross group boundaries, or does this only impact selections within a single group?

Reply:
Within a group, of course. Updated description